addons/skin.estuary/language/resource.language.en_gb/strings.po

```diff
@@ -743,3 +743,21 @@ msgstr ""
 msgctxt "#31163"
 msgid "Show Fanart background"
 msgstr ""
+
+#. Label of a setting that, if enabled, shows the name of the currently logged-in profile in the UI
+#: /xml/SkinSettings.xml
+msgctxt "#31164"
+msgid "Choose kind of profile identification"
```

Comment:
Show profile identifier

Reply:
True ;)

Will do, thx

Reply:
'Show' implies a on/off functionality. This label is for a choice: profile name, icon, ... So, "show" seems not correct to me.

Reply:
Think I agree with ksooo, so perhaps it could be "Choose profile identifier"

Reply:
Will use that ^^